sieve_of_Eratosthenes.py

```diff
@@ -0,0 +1,22 @@
+def sieve_of_eratosthenes(n):
+    
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
+++ b/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
@@ -1,5 +1,5 @@
 def sieve_of_eratosthenes(n):
-    
+
     prime = [True for i in range(n+1)]
     p = 2
     while (p * p <= n):
```